<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Overlap Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- 上传图像按钮 -->
   <widget class="QPushButton" name="button_upload_image1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Image 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_upload_image2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Image 2</string>
    </property>
   </widget>
   <!-- 透明度滑动条 -->
   <widget class="QSlider" name="slider_opacity">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <!-- 图像显示区域 -->
   <widget class="QLabel" name="label_image_overlap">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>700</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>Overlap Display</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <!-- 重置按钮 -->
   <widget class="QPushButton" name="button_reset">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <!-- 保存按钮 -->
   <widget class="QPushButton" name="button_save_overlap">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Overlap</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
</ui>
